wpilibcIntegrationTests/src/CircularBufferTest.cpp

```diff
@@ -5,12 +5,11 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "CircularBuffer.h"
+#include <array>
 
+#include "CircularBuffer.h"
```

Comment:
I'm trying to think of a clean way to skip include order checks for the integration tests, but I haven't so far.